app/modules/entities/components/layouts/infobox.svelte

```diff
@@ -53,58 +44,54 @@
     }
   }
   let waitingForEntities
-  $: if (displayedProperties) {
+  $: if (allowlistedProperties) {
     waitingForEntities = getMissingEntities()
   }
 
-  $: displayToggler = !shortlistOnly && withShortlist && entityPropertiesLonglist.length > longlistDisplayLimit
-  $: {
-    const claimsLonglist = _.pick(claims, propertiesLonglist)
-    entityPropertiesLonglist = Object.keys(claimsLonglist)
-    if (propertiesShortlist) {
-      const claimsShortlist = _.pick(claims, propertiesShortlist)
-      entityPropertiesShortlist = Object.keys(claimsShortlist)
-    }
-  }
-  $: {
-    if (withShortlist) {
-      if (showMore) {
-        displayedProperties = entityPropertiesShortlist
-      } else {
-        displayedProperties = entityPropertiesLonglist
-      }
-    }
-  }
+  let infoboxHeight, showDetails, infobox
+  const wrappedInfoboxHeight = 128
+  $: infoboxHeight = infobox?.clientHeight
+  $: wrappedSize = listDisplay && infoboxHeight > wrappedInfoboxHeight
```

Comment:
`infoboxHeight` might be undefined, so lets check explicitly for that case
```suggestion
  $: wrappedSize = listDisplay && infoboxHeight && infoboxHeight > wrappedInfoboxHeight
```

Reply:
done in b1594019e